Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
+- I contributed to building blocks or core technologies used by a variety of teams.
 
 </details>
 
 ## 🌱 Teaching and Mentoring
 
 - I am an expert mentor and teacher.
+- I am curious, listen, and encourage healthy debate.
+- I allow myself to be persuaded by great ideas.
 - I create clarity by communicating complex things in simple and accessible ways.
 - I understand my broadest impact comes from empowering those around me.
 - I am an expert at designing and reviewing software architecture and code.
-- I help teams and individuals understand our technical strategy, and the things we are doing to progress it.
+- I help teams and individuals understand our technical strategy, how it connects to our product strategy, and the things we are doing to progress it.
 - I regularly provide training and support for new methods, tools, and patterns.
-- I collaborate with managers to identify and find ways to fill any gaps in technical skills required in my teams.
-- I am curious, listen, and encourage debate.
-- I allow myself to be persuaded by great ideas.
+- I actively share knowledge relevant to multiple teams.
+- I collaborate with managers and lead software engineers to identify and find ways to fill any gaps in technical skills required in my teams.
 
 <details>
 <summary>Examples</summary>
 
 - I was consistently in demand for design and code review.
-- I am actively mentoring multiple Octopus engineers.
+- I became aware that I was a bottleneck and worked proactively with managers to uplift and empower other people to make the team more effective.
+- I am actively mentoring multiple Octopus engineers, including a Lead Software Engineer aspiring to become a Principal Software Engineer.
 - Engineers at Octopus seek out to work on my teams to learn from me.
 - After reviewing a code contribution from an engineer I saw could be improved, I worked directly with that engineer and their manager or team leader to uplift their skills in the identified growth area.
 - I helped a team make a course correction based on new information or ideas.
 - I provided appropriate technical freedom, and framing, that allowed the Lead Software Engineers on my teams to thrive independently.
+- I have elevated/shifted my focus significantly because the team(s) I support have been uplifted and empowered to the point they can deliver their mission with me as a sounding board.
 
 </details>
 
 ## 🧭 Culture and Leadership
 
-- I create an environment where others feel safe, by modeling vulnerability, thinking in the open, being wrong, and accepting corrections.
-- I generate energy within the teams I work with, motivating them to achieve great outcomes.
+- I am a proponent of [operational excellence](https://en.wikipedia.org/wiki/Operational_excellence) and [learning from incidents](https://www.learningfromincidents.io/about).
+- I get things right a lot of the time.
```

Comment:
```suggestion
```
I agree with what you're trying to get at here, but this feels like it risks misinterpretation, especially in 🧭 Culture and Leadership. I think being "right" is covered by 🦉 Domain expertise, and you've already added the qualification "[I am] wrong _sometimes_". I think that's clear enough.